### ISSUE TITLE
When windows are minimised, the window is still visible but not interact-able as if it's become part of the desktop wallpaper.

### ISSUE THREAD

**Issue by AMoonRabbit** [pop-os/shell#1140]
Distribution:
NAME="Pop!_OS"
VERSION="21.04"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 21.04"
VERSION_ID="21.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=hirsute
UBUNTU_CODENAME=hirsute
LOGO=distributor-logo-pop-os

Issue/Bug Description:
When minimising windows, a ghost version of the window remains in the background. Any interactions in the area of this ghost window result in interactions with the desktop area. Suspect this could be something to do with X11?

Expected behavior:
Minimising windows shoulld remove the visible window

Other Notes:
Seems to take a snapshot of the window as you can have an application full screen or in the snap mode and minimise it but the floating "restored" window will remain on the desktop

Screenshot from 2021-06-27 13-04-51
![image](https://user-images.githubusercontent.com/3766988/129432216-89d6e530-20b8-493d-83cf-ca439ec8f2d5.png)

This screenshot actually has no windows open, this is everything minimised

Relogged from as this is probably the better location for it. https://github.com/pop-os/pop/issues/1749

**Comment by leviport** [comment]
It looks like you're using non-standard themes and/or extensions. Does the same thing happen with the default theme and only the default extensions enabled?

**Comment by AMoonRabbit** [comment]
Yes, I have had it on a vanilla setup as well, but I will revert back to vanilla for a bit and see if I can get it to happen

**Comment by AMoonRabbit** [comment]
I have tested with the vanilla setup and still encountered the same issue. Also, noted that the windows that are no interactable are still active and appear live (Pop Shop UI was still animating as updates were progressing, yet the app was minimised but still visible due to the bug)

**Comment by AMoonRabbit** [comment]
Closing this as no further movement, but also haven't come across this issues since I've moved from a Surface Pro to a Framework laptop.
